ci/load-data.sh

```diff
@@ -1,28 +1,14 @@
-#!/usr/bin/env bash
+#!/bin/bash -e
 
 CWD="$(dirname "${0}")"
 
-declare -A argcommands=([sqlite]=sqlite
```

Comment:
currently if no backend is provided it will run for each all backend. I am curious, any reason for removing this?

Reply:
Before I deleted this piece of code, there were 2 places where I needed to define a list of backends that needed data loading. The problem is that after a change in one place, you need to remember to change in another. Makefile is essentially the entry point of Ibis, so I decided to leave the definition of the list in it.